src-docs/src/views/inline_edit/inline_edit_example.js

```diff
@@ -144,6 +152,27 @@ export const InlineEditExample = {
       snippet: inlineEditTitleSnippet,
       playground: inlineEditTitleConfig,
     },
+    {
+      title: 'Setting a placeholder',
```

Comment:
[copy suggestion] Placeholders should be used to inform users about the purpose of the input. In this case, for a slightly unusual input, I'd expect them to contain instructions. From a UX POV, I think we should encourage devs in that direction with our copy:

```suggestion
    {
      title: 'Setting placeholder instructions',
```